scripts/entrypoint.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
-cd /twofactorauth/site
-bundle config set path '/twofactorauth/vendor/cache'
+cd /twofactorauth
+bundle config set path '/app/vendor/cache'
 
```

Comment:
I building the regional sites doesn't work for me unless this is added:
```suggestion
bundle install --gemfile=/app/Gemfile

```

Reply:
I've made some modifications in 50f01f3 that should resolve this issue without having to re-install any gems

Reply:
It does indeed work now. Nice work! :clap: